R/interval.R

```diff
@@ -207,7 +207,8 @@ irregular <- function() {
 }
 
 unknown_interval <- function(x) {
-  if (is_empty(x)) FALSE else sum(x != 0) == 0
+  for(y in x) if(y) return(FALSE)
```

Comment:
Can you just do `for (y in x) return(as.logical(y))`?

Reply:
No, because the loop is for an early return.
If you immediately return as you've suggested it would only test if `interval(x)[[1]]!=0` not if `any(interval(x)[[i]]!=0)`

Reply:
Okay. Please add spaces to `for` and `if`.